.gitmodules

```diff
@@ -22,3 +22,6 @@
 [submodule "external/debugger"]
 	path = external/debugger
 	url = https://github.com/kevinb7/debugger
+[submodule "external/p5"]
+	path = external/p5
+	url = https://github.com/kevinb7/p5.js
```

Comment:
p5.js is quite a bit smaller than processing-js and is more modular.  I've made some pull requests (which have been accepted) to add a few things to make it easier to integrate.  There's one bug that needs fixing https://github.com/lmccart/p5.js/issues/492 and in general it needs more testing.  I'd like to see an image compare test suite similar to what processing-js has.